### ISSUE TITLE
Accessibility - Web - English - Basics Social Compliance System Page

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#125]
Why is there large graphic in the "Key Terms" section why text as part of the image instead of having individual graphics and text next to them? The ALT text for the graphic does not explain the cotton supply system at all. Unless I'm missing something in the text before or after this is not helpful to those using assistive technologies like screen readers.

Going to create an issue for Spanish and French too.

Please see the attached screenshot.

![complychain - english - bascis of social compliance system page - poor alt text](https://user-images.githubusercontent.com/10423487/44423305-c9203580-a553-11e8-9597-db113c7600de.png)
